Makefile

```diff
@@ -56,6 +56,13 @@ clean:
 compile:
 	${SOURCE_DIR}/build/build.sh
 
+################################################################################
+# runs all the tests
+################################################################################
+.PHONY: test
+test:
+	go test ./...
```

Comment:
Should add `-mod=vendor` to this, to ensure vendor is up to date too.

Reply:
Thanks. Fixed.